.circleci/config.yml

```diff
@@ -52,10 +52,10 @@ jobs:
           command: npm cache clean --force
       - run:
           name: "Install dependencies"
-          command: npm ci
+          command: npm ci --force
       # Cache node_modules across different internal jobs in the workflow and across different circleci runs
```

Comment:
Are we going to change this to force?

Reply:
It's already changed. Please check the latest change